public/lib/column-defaults.js

```diff
@@ -34,11 +34,24 @@ import publicationLocationTemplate from "components/content-list-item/templates/
  *      colspan: number, // colspan of this field
  *      title: string, // title attribute contents for the column heading
  *      templateUrl: string // URL for the content-list-item template for this field
+ *      isSortable: boolean // Can the column be sorted by clicking its header?
+ *      sortField: string // The field to sort on, if different from `name`. Can be an item path, e.g. `a.nested.field`
  * }
  */
 
 var templateRoot = '/assets/components/content-list-item/templates/';
 
+const createSortTemplate = (sortField, labelHTML) => `
+    <div ng-click="toggleSortState('${sortField}')" class="content-list-head__heading-sort-by">
```

Comment:
Would `content-list-head__heading--sort-by` be more BEMy?

Reply:
As I understand it, the M of BEM is for flags – things that change state. I was considering `heading-sort-by` to be an element of the `content-list-head` block – it's not a subelement of `heading` in BEM-land. Although it _is_ a subelement of heading in the DOM. 😁

So I suppose it depends how flat you'd like the hierarchy to be – I've followed some docs in the past that suggest that [flattening elements isn't necessary](http://getbem.com/faq/) (see 'What would be a class name for an element inside another element?'), but it's fair to say tastes may differ, lmk 👍

Reply:
`heading-sort-by` 👍 

This was a minor comment, please feel free to ignore too!